### DISCUSSION TITLE
Security Onion Certification available now!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#5887] [Announcements]
https://blog.securityonion.net/2021/10/security-onion-certification-available.html

![Screen Shot 2021-10-14 at 10 27 03 AM](https://user-images.githubusercontent.com/1659467/137357233-f41ad3b4-4d36-4cd0-99be-c158b4ec4baa.png)

**Comment by AnyrunMalware** [comment]
Nice! Just a note: the link for the book doesn't seem to work, when I click on it, it just brings me to the beginning of the same site

Security Onion Documentation Paperback book (Available online here)

**Reply by rwwiv** [reply]
Thanks for pointing that out @AnyrunMalware, that link is now fixed.
